attestation-agent/attester/src/nvidia/mod.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2025 Confidential Containers
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::{Attester, TeeEvidence};
+use anyhow::{Context, Result};
+use base64::Engine;
+use nvml_wrapper::{enums::device::DeviceArchitecture, Nvml};
+use serde::Serialize;
+
+pub fn detect_platform() -> bool {
+    // TODO: loop all devices here to check their CC status?
```

Comment:
Do we have answer for this?

Reply:
No but I think a check for "CC capable" would be sufficient here. WDYT?

Reply:
Yes. We do not need to return true if the GPUs/devices are not confidential

Reply:
I have this covered. `nvml-wrapper` just failed for this but it's fixed now. Will push an update to this shortly.